src/openforms/prefill/contrib/customer_interactions/tests/test_plugin.py

```diff
@@ -283,3 +288,149 @@ def test_prefill_values_no_channel(self):
         state = submission.load_submission_value_variables_state()
 
         self.assertEqual(state.variables["empty-preferences"].value, {})
+
+    def test_create_multiple_prefill_variables_for_same_component(self):
```

Comment:
All these tests below are testing the new DB constraint for `FormVariable` model, right? I think they belong to `src/openforms/forms/test/variables/test_model.py`